apps/vr-tests/src/stories/Popover.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import { Popover, PopoverTrigger, PopoverSurface } from '@fluentui/react-popover';
+import { PositioningShorthandValue } from '@fluentui/react-positioning';
+
+const ExampleContent = () => {
+  return (
+    <div>
+      <h3>Popover content</h3>
+
+      <div>This is some popover content</div>
+    </div>
+  );
+};
+
+const positions: PositioningShorthandValue[] = [
```

Comment:
```suggestion
const positions: PopoverProps['positoning'][] = [
```

Can we use `PopoverProps`? Then we will not need to have `@fluentui/react-positioning` in deps of `vr-tests`

Reply:
Updated in [959bd37](https://github.com/microsoft/fluentui/pull/20541/commits/959bd3754f75119cdc36fa561e00fdb7971f0a4a)